.gitignore

```diff
@@ -1 +1,2 @@
-.idea/
\ No newline at end of file
+.idea/
+node_modules/
```

Comment:
Always add newline at the end

Reply:
Can you add also `.vscode` and `.history/`, for people who use VS Code as their editor?